tests/test_samples.py

```diff
@@ -108,8 +108,24 @@ def test_print_shortcut_target_readable(self):
 
         our = mock_stdout.getvalue()
 
-        json_path = os.path.join(JSON_DIR, f"shortcut_target_only.txt")
-        with open(json_path, 'r') as fp:
+        txt_path = os.path.join(JSON_DIR, "shortcut_target_only.txt")
+        with open(txt_path, 'r') as fp:
+            their = fp.read()
+
+        self.assertEqual(our, their)
+
+    def test_print_unknown_target(self):
+        with open('tests/samples/unknown_target', 'rb') as indata:
```

Comment:
Could you please add tests for [sample](https://www.virustotal.com/gui/file/891b561432a26bd71bb67dce82d52bc99e3b45c2d2bf6b714d496ef62b40f2cb) with `Unknown` block?

Reply:
I'm sorry as it may seem very picky, but I am not supposed to reupload VT samples as they are not free access. 🙁
I have found [this](https://bazaar.abuse.ch/sample/5a3f1d14b9cc4890db64fbc41818d7039f25b0120574dcdec4e20d13e6b2740c) and [this](https://bazaar.abuse.ch/sample/050c65d45e5f21018aa940f0188c4aa1318ac3df865d901f8643ed7ce4a4b52c) sample on Malware Bazaar, which is free for access, but both are 4MB instead of VT's original one at 800KB. I don't know if you have a tool that could be used to forge a sample that would be smaller for the tests? On top of it, I think it's a good idea to have as little real malicious samples stored in a non-defanged state on a public access platform. Let me know what you think! 🙂

Reply:
Well, I understand your point. Luckily, we can easily corrupt/modify any of test samples. I'll do it once this PR is merged.

Here is output from corrupted `invalid_date2`:
```
EXTRA BLOCKS:
    SPECIAL_FOLDER_LOCATION_BLOCK
        Size: 16
        Special folder id: 37
        Offset: 213
    UNKNOWN_BLOCK
        Block:
            Size: 28
            Extra data sha256: 4f022b4bc7668d870e158010c9877224df1b9b07bd24ef32b731963232b15eb2
            Size: 153
            Extra data sha256: 40ebe6ee6ad3303ae2db241742e857aee523b21553a51e2f6c2a2461f1010879

```
I'll fix print of "Block"  afterwards.